src/App.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import axios from 'axios';
 import Overview from './components/overview/Overview.jsx';
 import Reviews from './components/reviews/Reviews.jsx';
-import Questions from './components/questions/Questions.jsx';
+import QuestionsAnswers from './components/questions/QuestionsAnswers.jsx';
```

Comment:
Possibly verbose, but not necessarily a problem.

Reply:
its really for me because i will have an actual Question component, so to make no mistake between Questions/Question I figured QuestionAnswers was better for the parent component

Reply:
Makes sense. 👌